packages/eas-json/src/build/schema.ts

```diff
@@ -39,10 +40,12 @@ const AndroidBuildProfileSchema = CommonBuildProfileSchema.concat(
     ),
 
     artifactPath: Joi.string(),
+    applicationArchivePath: Joi.string(),
+    buildArtifactsPaths: Joi.array().items(Joi.string()),
```

Comment:
I think you originally wanted to make it `string | string[]`, Did you change your mind?

Reply:
@brentvatne 
<img width="1100" alt="Screenshot 2022-08-31 at 14 48 04" src="https://user-images.githubusercontent.com/5256730/187681862-794bd0ea-1e55-40e6-aed8-6b5b7456f08b.png">